.github/workflows/e2e.yml

```diff
@@ -379,7 +381,7 @@ jobs:
           retention-days: 4
       - name: Upload build number
         uses: actions/upload-artifact@v4
-        if: ${{ matrix.project != 'skip' && steps.affected.outputs.affected == 'true' }}
+        if: ${{ matrix.project != 'skip' }}
```

Comment:
The `e2e-affected` action now generates this file so we can have the artifact to verify with.